tools/functional-tester/etcd-agent/agent.go

```diff
@@ -75,23 +77,59 @@ func (a *Agent) start(args ...string) error {
 	return nil
 }
 
+var alreadyFinished = errors.New("os: process already finished")
```

Comment:
This is not defined in os package, which just returns `errors.New("os: process already finished")` in it.